server/fastAPI/quiz.py

```diff
@@ -0,0 +1,43 @@
+from fastapi.encoders import jsonable_encoder
+from pydantic import BaseModel, validator
+from typing import Optional
+import base64
+import xmltodict
+
+from question import OwnerInfo
+
+
+# model definition
+class Quiz(BaseModel):
+    owner: OwnerInfo
+    is_simulation: Optional[bool] = False
```

Comment:
We can't have a required boolean as optional, we need it to filter queries. Is this variable in the MoodleXML file or should we add a checkbox in the frontend?
```suggestion
    is_simulation: bool = False
```